core/src/main/java/hudson/model/ListView.java

```diff
@@ -109,6 +111,14 @@ public ListView(String name, ViewGroup owner) {
         this.owner = owner;
     }
 
+    /**
+     * Sets the columns of this view.
+     */
+    @DataBoundSetter
```

Comment:
I suppose this is intended to allow list views to be configured with a `columns` parameter from a script.